client/src/component/Codewars.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import CodeWarsCard from "/CodeWarsCard";
+
+const TraineeDashboard = () => {
```

Comment:
The file name and export should match in this example eg either Codewars or TraineeDashboard

Reply:
i think you can change the name of file and export to dashboard

Reply:
Done